apps/laboratory/src/utils/DataUtil.ts

```diff
@@ -61,6 +66,11 @@ export const ethersSdkOptions = [
     title: 'Email and Social',
     link: '/library/ethers-email/',
     description: 'Configuration using ethers and implementing email and social login'
+  },
+  {
```

Comment:
What about `ethers5` no email?

Reply:
We could even test solana since it's enabled for Solana as well

Reply:
Removing ethers5 email until we get it working properly

Let me change this for solana

Reply:
It doesn't really have a working email tbh. LEaving it out until we have it stable. 
Adding solana too 🫡